src/components/forms/AddItemForm.tsx

```diff
@@ -85,7 +85,10 @@ export function AddItemForm({ listPath, data: unfilteredListItems }: Props) {
 		}
 	};
 	const navigateToListPage = () => {
-		navigate("/list");
+		if (listPath) {
+			const listName = listPath.split("/").pop();
```

Comment:
praise: this is a really cool way to get the `listName` from the path and a cool way to use `split` and `pop`

Reply:
shout out to my  past life of freeCodeCamp  tutorial rabbit holes!